litmus-portal/frontend/src/views/Analytics/Overview/index.tsx

```diff
@@ -308,11 +308,11 @@ const Overview: React.FC = () => {
       )}
       {applicationDashboardCount > 0 && (
         <RecentOverviewContainer
-          heading="Recent Application Dashboards"
+          heading="Recently viewed application dashboards"
           buttonLink="/analytics/dashboard/create"
           buttonImgSrc="./icons/cloudWhite.svg"
-          buttonImgAlt="Add dashboard"
-          buttonText="Add dashbaord"
+          buttonImgAlt="Create dashboard"
+          buttonText="Create dashbaord"
```

Comment:
```suggestion
          buttonImgAlt="Create a dashboard"
          buttonText="Create a dashbaord"
```

Reply:
we have removed a from the text @vanshBhatia-A4k9 during design updated and UX fixes